osf_tests/factories.py

```diff
@@ -257,10 +257,14 @@ class InstitutionFactory(DjangoModelFactory):
     email_domains = FakeList('domain_name', n=1)
     orcid_record_verified_source = ''
     delegation_protocol = ''
+    can_request_access = False
 
     class Meta:
         model = models.Institution
 
+    @classmethod
+    def _create(cls, *args, **kwargs):
```

Comment:
I don't know what the point of doing this is, write a comment/docstring explaining or remove it.